src/bundle/ui-dev/src/modules/multi-file-upload/components/drop-area/drop.area.component.js

```diff
@@ -61,10 +61,9 @@ export default class DropAreaComponent extends Component {
         return (
             <form className="c-drop-area" multiple onDrop={this.handleUpload}>
                 <div className="c-drop-area__message c-drop-area__message--main">{dropActionMessage}</div>
-                <div className="c-drop-area__btn-select" onClick={this.openFileSelector} tabIndex="-1">
-                    <Icon name="upload" extraClasses="c-drop-area__icon ez-icon--light ez-icon--small-medium" />
+                <button class="btn btn-primary c-drop-area__btn-select" onClick={this.openFileSelector} tabIndex="-1">
```

Comment:
please add `type="button"` to prevent submiting the form if it will be nested inside a form